examples/skysync-cli-example-create-job-01.ps1

```diff
@@ -0,0 +1,77 @@
+<#
+.DESCRIPTION
+    Sample script in PowerShell to showcase the use of the CLI to generate and manage SkySync jobs. Example scenario:
+        * Create an NFS connection
+        * Create a Box connection
+        * Create a SkySync job that utilizes these connections
+        * Start the job
+
+.REFERENCE
+    https://skysync.atlassian.net/wiki/spaces/SSP/pages/15302683/SkySync+Command-line+Interface
+
+.NOTES
+    Setup/Config:
+        1. Create file 'skysync-cli.json' and place one directory above cli project/script that is being executed and add the following JSON block to the file:
+
```

Comment:
i think we should create the file and this step should say that the user should update it.

Reply:
Agreed, because this file can live in the same directory. It doesn't have to be one directory above.

Reply:
created file, note had to force the commit for the skysync-cli.json since this file was specifically ignored